duckduckgo/iOS
# Fix Babel-Pollyfil page break

Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/465549777224334
CC:

**Description**:

Include smaller JS files for iOS 10+ and modify the ABP JS for iOS 9 to not include babel pollyfil if already detected.

**Steps to test this PR**:

iOS 9:

1. Go to reddit.com/r/duckduckgo and confirm you can interact with the page.  Check the console to confirm easylist blocking.
1. Go to Evans Cycles and check the console to confirm easylist blocking in the console.

iOS 10+:

1. Go to reddit.com/r/duckduckgo and confirm you can interact with the page.  Check the console to confirm easylist blocking.
1. Go to Evans Cycles and confirm easylist blocking in the console.
1. Using the web console check which user-scripts have been loaded.  The smaller ABP-filter-parser (~1000 lines) should be the one in use and the smaller Bloom filter (~250 lines) should be the ones in use. (Bear in mind the bloom filter js is only loaded after the easy list is cached)


###### Reviewer Checklist:


###### PR DRI Checklist:
